source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Enhance import-export with OVA
+category: feature
+authors: arik
+feature_name: Enhance import-export with OVA
+feature_modules: engine, vdsm
+feature_status: planning
+---
+
+# Backgroud
+Open Virtual Appliance (OVA) is a package that contains the different elements of a virtual machine (VM). OVA is typically a tar archive file that includes the configuration of the VM in the form of the Open Virtualization Format (OVF) as well as virtual disks and other VM-related data.  
+
+OVA is a known format that is used by different organizations and virtualization platforms. Its specification can be found [here](http://www.dmtf.org/standards/ovf). The standard provides an ["open, secure, portable, efficient and extensible format for the packaging and distribution of software to be run in virtual machines"](https://en.wikipedia.org/wiki/Open_Virtualization_Format).  
+
+Recently, oVirt was enhanced with support for importing OVAs that were generated by VMware. oVirt also makes extensive use of the OVF standard to describe VM configurations, such as VM snapshots and exported VMs.  
+
+The ability to package all the VM data, optionally compressed, in one file eases the processes of backing up a VM and porting a VM to a different platform or hypervisor. Unfortunately, the portability is limited by compatibility issues. Different disk formats, drivers, guest-agents, and OVF structure, hinder the ability to import any OVA to any platform. Therefore, different organizations develop their own conversions tools, like Red Hat's [virt-v2v](http://libguestfs.org/virt-v2v.1.html), for converting VMs from other platforms to their platform.   
+
+# Scope
+This feature is about enhancing the import VM and export VM processes in oVirt by making a broader use of the OVA format.  
+
+## Upload OVA
+Today, the process of importing an OVA is limited to OVA files compatible with those produced by VMware and requires one to make the OVA file accessible by  one of the hosts managed by oVirt with the right file permissions (vdsm:kvm). It is a cumbersome and error-prone process.  
+
+This feature provides a broader and more user-friendly alternative that enables one to upload a local OVA file straight from a client (browser/rest-api based, much like the upload-image functionality that was recently added to oVirt).  
+
+It would be possible to upload OVA files that were generated either by oVirt or by external sources. Officially, the only external source being supported is VMware. However, in practice, the process could work with OVA files generated by other source as long as their OVF configuration contains the data oVirt consumes (in the expected form) and virt-v2v knows how to convert them.
+
+## Export as OVA
```

Comment:
Suggested edits:
Suggested edit: "This feature provides a broader and more use-friendly alternative that will enable you to upload a local OVA file directly from the client"
comment: 2nd half a bit unclear. Is the parenthesis in the right location?